appc/appconnector.go

```diff
@@ -249,6 +295,18 @@ func (e *AppConnector) Wait(ctx context.Context) {
 	e.queue.Wait(ctx)
 }
 
+// Close closes the connector and cleans up resources associated with it.
+// It is safe (and a noop) to call Close on nil.
+func (e *AppConnector) Close() {
+	if e == nil {
+		return
+	}
+	e.mu.Lock()
+	defer e.mu.Unlock()
+	e.queue.Shutdown() // TODO(creachadair): Should we wait for it too?
+	e.pubClient.Close()
```

Comment:
don't have to close updatePub and storePub? (not documented, related to my other Slack message on the subscribe side)

Reply:
They are closed by the closure of the client, see https://pkg.go.dev/tailscale.com/util/eventbus#Client.Close